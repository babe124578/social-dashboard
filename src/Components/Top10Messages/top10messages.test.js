import React from 'react';
import Top10Messages from "./top10messages.js";
import { shallow } from 'enzyme';
import shallowRenderer from 'react-test-renderer/shallow'

it('shallow snapshot', () => {
    const renderer = new shallowRenderer()
    const snapshot = renderer.render(<Top10Messages />)
    expect(snapshot).toMatchSnapshot()
});

it('should render without crashing', () => {
    shallow(<Top10Messages />);
});