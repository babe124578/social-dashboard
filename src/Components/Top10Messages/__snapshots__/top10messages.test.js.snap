// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow snapshot 1`] = `
<ForwardRef
  fluid={true}
>
  <ForwardRef
    noGutters={false}
  >
    <ForwardRef>
      <ForwardRef
        animation={true}
        autohide={false}
        delay={3000}
        show={true}
        transition={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "defaultProps": Object {
              "appear": false,
              "in": false,
              "mountOnEnter": false,
              "timeout": 300,
              "unmountOnExit": false,
            },
            "displayName": "Fade",
            "render": [Function],
          }
        }
      >
        <ForwardRef
          closeButton={false}
          closeLabel="Close"
          id="headertoast"
        >
          <strong
            className="mr-auto"
            id="message"
          >
            Message
          </strong>
          <small
            id="engagement"
          >
            Engagement
          </small>
        </ForwardRef>
      </ForwardRef>
    </ForwardRef>
  </ForwardRef>
</ForwardRef>
`;
